route/errors.go

```diff
@@ -77,22 +77,12 @@ func (r *Router) handlerReturnWithError(w http.ResponseWriter, he handlerError,
 
 func (r *Router) handleOTLPFailureResponse(w http.ResponseWriter, req *http.Request, otlpErr husky.OTLPError) {
 	r.Logger.Error().Logf(otlpErr.Error())
-
-	if otlpErr != husky.ErrInvalidContentType {
-		err := husky.WriteOtlpHttpFailureResponse(w, req, otlpErr)
-		if err != nil {
-			// If we made it here we had a problem writing an OTLP HTTP response
-			resp := fmt.Sprintf("failed to write otlp http response, %v", err.Error())
-			r.Logger.Error().Logf(resp)
-			w.Header().Set("Content-Type", "text/plain")
-			w.WriteHeader(http.StatusInternalServerError)
-			_, _ = io.WriteString(w, resp)
-		}
-	} else {
-		// Since we have an invalid content type for an OTLP HTTP response we choose to use to text/plain
+	if err := husky.WriteOtlpHttpFailureResponse(w, req, otlpErr); err != nil {
```

Comment:
is the error here always a content type error?

Reply:
No, API key missing or invalid errors are also possible to come through here

Reply:
right, the description says it will use plain for content type errors, but this changes the behaviour for all errors?